public/models/game/game.js

```diff
@@ -59,6 +59,18 @@ var Game = Map.extend({
 				newVal.__listSet = {gameId: this.attr("id")};
 				return newVal;
 			}
+		},
+		videoUrl: {
+			serialize: true,
+			get: function (setVal) {
```

Comment:
nm ... I forgot that serialize calls `.get` internally.  Still, I would probably solve this with a setter.  That way the stored value is always the right value.